src/main/java/com/thebois/models/beings/Colony.java

```diff
@@ -29,30 +24,39 @@
     private final Inventory inventory = new Inventory();
 
     /**
-     * Initializes with already existing beings.
+     * Creates a colony with the pawns that belong to it.
      *
-     * @param beings The beings that should belong to the colony
+     * @param beings Pawns to be associated with the colony.
      */
     public Colony(final Collection<IBeing> beings) {
         setBeings(beings);
     }
 
     /**
-     * Creates an instance of Colony with a number of pawns.
+     * Returns the players' colony.
      *
-     * @param vacantPositions Positions in the world that a Pawn can be placed on.
-     * @param world           The world the pawns move around in.
+     * @return the colony.
      */
-    public Colony(final Iterable<Position> vacantPositions, final IWorld world) {
-        createBeings(vacantPositions, world);
+    public Colony getColony() {
+        return this;
     }
```

Comment:
This is still unnecessary, right? Since you don't need to call `colony.getColony()` if you already have the colony.

Reply:
Omg....
Lets not talk about this....